File: app/src/main/java/com/example/myapplication/activities/mangapage/read_manga_mvp/ReadMangaInterface.java
package com.example.myapplication.activities.mangapage.read_manga_mvp;

import com.example.myapplication.models.mangamodels.ReadMangaModel;

import java.util.List;

public interface ReadMangaInterface {
    void onGetMangaContentDataSuccess(ReadMangaModel mangaContents);

    void onGetMangaContentDataFailed();

    void onGetMangaChaptersDataSuccess(List<ReadMangaModel.AllChapterDatas> allChapters);

    void onGetMangaChaptersDataFailed();
}

File: app/src/main/java/com/example/myapplication/activities/mangapage/read_manga_mvp/ReadMangaInterface.kt
package com.example.myapplication.activities.mangapage.read_manga_mvp

import com.example.myapplication.models.mangamodels.ReadMangaModel

interface ReadMangaInterface {
    fun onGetMangaContentDataSuccess(mangaContents: ReadMangaModel)

    fun onGetMangaContentDataFailed()

    fun onGetMangaChaptersDataSuccess(allChapters: List<ReadMangaModel.AllChapterDatas>)

    fun onGetMangaChaptersDataFailed()
}
